custom_components/powercalc/data/sengled/E11-N13_E11-N13A_E11-N14_E11-N14A/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "shellyplugus-d4d4da091468",
```

Let's use some sanitized name for the smart plug, that will make it easier to list and group the used smart plugs when we'd like to make an overview in the future.

```suggestion
    "measure_device": "Shelly Plug US",
```